commit 3c931e6454804db1070ad9a094a521ddf6023ab2
Author: CQ Tang <cq.tang@intel.com>
Date:   Mon Jan 30 09:39:52 2017 -0800

    iommu/vt-d: Fix some macros that are incorrectly specified in intel-iommu
    
    commit aaa59306b0b7e0ca4ba92cc04c5db101cbb1c096 upstream.
    
    Some of the macros are incorrect with wrong bit-shifts resulting in picking
    the incorrect invalidation granularity. Incorrect Source-ID in extended
    devtlb invalidation caused device side errors.
    
    To: Joerg Roedel <joro@8bytes.org>
    To: David Woodhouse <dwmw2@infradead.org>
    Cc: iommu@lists.linux-foundation.org
    Cc: linux-kernel@vger.kernel.org
    Cc: CQ Tang <cq.tang@intel.com>
    Cc: Ashok Raj <ashok.raj@intel.com>
    
    Fixes: 2f26e0a9 ("iommu/vt-d: Add basic SVM PASID support")
    Signed-off-by: CQ Tang <cq.tang@intel.com>
    Signed-off-by: Ashok Raj <ashok.raj@intel.com>
    Tested-by: CQ Tang <cq.tang@intel.com>
    Signed-off-by: Joerg Roedel <jroedel@suse.de>
    Signed-off-by: Greg Kroah-Hartman <gregkh@linuxfoundation.org>

Index: linuxtest/dummy/rpi_1247_3c931e6454804db1070ad9a094a521ddf6023ab2.txt
===================================================================
--- /dev/null
+++ linuxtest/dummy/rpi_1247_3c931e6454804db1070ad9a094a521ddf6023ab2.txt
@@ -0,0 +1 @@
+dummy file to ensure patch has content.
